include/oneapi/tbb/detail/_export.h

```diff
@@ -19,28 +19,30 @@
 
 #if defined(__MINGW32__)
     #define _EXPORT __declspec(dllexport)
-#elif defined(_WIN32) || defined(__unix__) || defined(__APPLE__) // Use .def files for these
+#elif defined(_WIN32)
     #define _EXPORT
+#elif defined(__unix__) || defined(__APPLE__) // Use .def files for these
+    #define _EXPORT __attribute__ ((visibility ("default")))
```

Comment:
Rename `_EXPORT` to something `__TBB_*` to avoid possible conflicts with user code